components/display/images.rst

```diff
@@ -0,0 +1,122 @@
+.. _display-image:
+
+Images
+======
+
+.. seo::
+    :description: Instructions to display static images on ESPHome
+    :image: image-outline.svg
+
+Use this component to store graphical images on the device, you can then draw the images on compatible displays.
+
+.. code-block:: yaml
+
+    image:
+      - file: "image.png"
+        id: my_image
+        resize: 100x100
+
+.. code-block:: yaml
+
+    image:
+      - file: mdi:alert-outline
+        id: alert
+        resize: 80x80
+
+.. code-block:: yaml
+
+    image:
+      - file: https://esphome.io/_images/logo.png
```

Comment:
This URL is part of an example, and as such must stay absolute; even if it contains esphome.io